### PR TITLE
Add K8S service sample

### PR DESCRIPTION
**Pull request by baixinsui**
#187

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,142 +33,184 @@ state of your service, interacting with the fundamental APIs:
...
...H4YuMrZf6hhOQ9D/WoYe9RFD538+9bbfu6OpD2B15PhoZJ7Ul+S3XP02hFrEYyOq9NrH46xXCWxyL+q4thGnBR7x0FTDZEBqVpcHa0AqggxAEYkF7k3+i71uE7k/a6OqjK4YOv+i92/+Hp/Rv86tvf15MQLP0sb1MYr8LxV8WLp3mkDAnSKMaDrevR62q5t2ODe6z6PLqIAndt5wKvCjjm+UYqfeUEa4Pqr5iPu3gPlTjyVx+yVbjp4Thk4X1COkVvPD0SwzByhaL0/3nYB2xrmiuJRJ4GRJ3WPo9bnvURR21+b66lp7taeI4G0ifuGZwPFauvWgs6G6hQuAu7+rf4tCxw91TIf9VlMSetVabz0PzxrcJdaBeaJZOjzeg71yns67IIL/bNKOKhDi4phQxrnXWQ5Go8NqNO9OF+TEpMy7xPyBtjGJLZNSHwgGP5XmJ45gzYm6ZKjGICRVpnUPvUN70ZQmZRKAdZsbLzbcAirCIxzAQEV5UiZBF3RA7s0r9dknFtzjH7DmMRlduPKgCXGwP3ygzpFpjBJJBLmccFiilEkQQRKsk9svxxs1jwSxbNUNo6c8Q56t2zxCuzmeinqk+dQyKTQmzBTELmNjhFBirhfb7spbVuqSvgF8GRjYLalHjhe4Bk6EkMdof+Uqw1leBM2FjX8EwzDXG45CCwVrhdlTpEwCa7mxEDJ0NmQvCO9auk0Bx/SpBS1ty1F85PTDW45KeLjAbW0+1FSGspFK7RJ/OzmHeV+GxLdNrFqi98+TMgks0iZJ6s4QSGYkA1j6qdHnSZkEtr6+tDOrN9pq91kH0MWxUa8szUz+VCmTwCZtCMPU0dWiUHA2qSwvuWP/V8rmZobKk+scCr/XaBsg1fflSXDMUyoPz5cyCdzSXEoDzfkYZw44dBEvEfJDU09G8IJNxJn9kUpFuc/Kg5DK2XkyHBOyikPXE/u5+GVpQ7Uv2jwV4PdfSgHO+eJvIdR+kUIM6rxaOnqOOSEF8NtSJoHhVaDygdweOioRo9+RNr+oy6QAfZehEDJr+p+JWjnv29sJfI2USdDTdMorGa+SMgla2yefxaxCreejPrfY5IWBuWM/4qegN+X0GecxbmuwyJUZWz5ahy31UbnwKQnnfwh9FwSVstH6RXv3jTfeeOONN95444033iiK/wDpWyMpdIpCEgAAAABJRU5ErkJggg==
```
Comment (iskey) [inline README.md:47]: Can we keep it shorter in Readme?

File: samples/K8S-cluster.yaml [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+    }
+
+    variable "flavor_flavor_id" {
+    type = string
+    description = "The flavor id of all nodes in the K8S cluster."
+    }
+
+    variable "network_id" {
+    type = string
+    default= "6a7c406d-5bc6-4b50-8f75-948aea69cb24"
```
Comment (iskey) [inline samples/K8S-cluster.yaml:118]: please avoid using a specific id here.

File: samples/K8S-cluster.yaml [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+    resource "huaweicloud_compute_instance" "k8s-node" {
+    count = var.flavor_worker_nodes_count
+    name = "k8s-node-${count.index}"
+    availability_zone = data.huaweicloud_availability_zones.osc-az.names[0]
+    flavor_id = var.flavor_flavor_id
+    network {
+        uuid = var.network_id
+    }
+
+    security_group_ids = [ "668833c8-459d-4d15-871d-34c3cbcda9d4" ]
```
Comment (iskey) [inline samples/K8S-cluster.yaml:158]: Please also fix this.

File: README.md [hunk 4]
```diff
@@ -33,142 +33,184 @@ state of your service, interacting with the fundamental APIs:
...
-   currency: euro
-# The flavor of the service, the @name/@version/@flavor can locate the 
-specific service to be deployed.
+  # The business model(`flat`, `exponential`, ...)
+  model: flat
+  # The rental period (`daily`, `weekly`, `monthly`, `yearly`)
+  period: monthly
+  # The billing currency (`euro`, `usd`, ...)
+  currency: euro
+# The flavor of the service, the @name/@version/@flavor can locate the specific service  to be
```
Comment (iskey) [inline README.md:61]:
```suggestion
# The flavor of the service, the @category/@name/@version/@flavor can locate the specific service to be 
```

File: README.md [hunk 5]
```diff
@@ -33,142 +33,184 @@ state of your service, interacting with the fundamental APIs:
...
+
+    variable "network_id" {
+    type = string
+    default= "6a7c406d-5bc6-4b50-8f75-948aea69cb24"
+    description = "The network id of all nodes in the K8S cluster."
+    }
+
+    variable "security_group_id" {
+    type = string
+    default= "668833c8-459d-4d15-871d-34c3cbcda9d4"
```
Comment (iskey) [inline README.md:160]: please try to replace all the ids with `data resources` as below.

**Review by iskey — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (baixinsui) [reply README.md:47]: Ok. I will find a smaller icon to replace it.

File: samples/K8S-cluster.yaml (hunk 2 above)
Comment (baixinsui) [reply samples/K8S-cluster.yaml:118]: Ok.

File: samples/K8S-cluster.yaml (hunk 3 above)
Comment (baixinsui) [reply samples/K8S-cluster.yaml:158]: Ok.

File: README.md (hunk 5 above)
Comment (baixinsui) [reply README.md:160]: Ok.

File: samples/K8S-cluster.yaml [hunk 6]
```diff
@@ -0,0 +1,176 @@
...
+
+    variable "network_id" {
+    type = string
+    default= "6a7c406d-5bc6-4b50-8f75-948aea69cb24"
+    description = "The network id of all nodes in the K8S cluster."
+    }
+
+    variable "security_group_id" {
+    type = string
+    default= "668833c8-459d-4d15-871d-34c3cbcda9d4"
```
[the remark this answers is not in the document]
Comment (baixinsui) [reply samples/K8S-cluster.yaml:124]: Ok.

**Review by iskey — APPROVED** [review]
looks good.

**Pull request merged**